packages/documentation/gatsby-config.js

```diff
@@ -66,5 +66,35 @@ module.exports = {
     `gatsby-transformer-remark`,
     `github-api-pages`,
     `mdx-pages`,
+    {
+      resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+      options: {
+        // Fields to index
+        fields: [`title`, `tags`],
+        // How to resolve each field`s value for a supported node type
+        resolvers: {
+          // For any node of type MarkdownRemark, list how to resolve the fields` values
+          SitePage: {
+            title: (node, getNode) => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.title) {
+                return node.context.frontmatter.title;
+              } else {
+                console.info('Page title missing from front matter.', node);
```

Comment:
I think we should have some warning for pages that are missing frontmatter- they won't be indexed without this.